src/styles/index.css

```diff
@@ -0,0 +1,111 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  padding: 32px 19px 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 150%;
+  color: #060b35;
+  padding-bottom: 4px;
+  padding-left: 16px;
+  padding-right: 16px;
+  max-width: 250px;
```

Comment:
The `.card__title` class has a `max-width` property that exceeds the width of the card itself. Ensure that the `max-width` does not exceed the parent container's width to prevent layout issues.